src/main/scala/com/github/unchama/seichiassist/menus/minestack/MineStackSelectItemColorMenu.scala

```diff
@@ -47,11 +48,10 @@ case class MineStackSelectItemColorMenu(mineStackObj: MineStackObj) extends Menu
         )
       )
     )
-    IO {
-      MenuSlotLayout(buttonMapping.map {
-        case (index, button) => index -> button.unsafeRunSync()
-      }: _*)
-    }
+    for {
+      mapping <- buttonMapping.map(_.sequence).sequence
```

Comment:
`map(f).sequence == traverse(f)`です
```suggestion
      mapping <- buttonMapping.traverse(_.sequence)
```

Reply:
:+1:

Reply:
ありがとうございます！
修正しました！
6ff2959127a7a8c7387b225ae3f76a59792df398